internal/platform/time_notcgo.go

```diff
@@ -1,7 +1,21 @@
-//go:build !cgo
+//go:build !cgo && !windows
 
 package platform
 
+import "time"
+
 func nanotime() int64 {
 	return nanotimePortable()
 }
+
+// nanoBase uses time.Now to ensure a monotonic clock reading on all platforms
+// via time.Since.
+var nanoBase = time.Now()
```

Comment:
mind moving this back to time.go? I think testing is less error-prone in the package that doesn't have flags. It shouldn't cause harm when not used, right?